### PR TITLE
[ios] Remove destroy

### PR DESCRIPTION
**Pull request by dcvz**
Missed before the 3.0.0 release

### CODE REVIEW

**Comment by jspizziri** [comment]
@dcvz We'd need to remove the method from JS too.

**Review by jspizziri — DISMISSED** [review]
LGTM, I don't have all the perspective but the docs and Js seem to check out.

**Comment by mpivchev** [comment]
Also attempts to fix https://github.com/doublesymmetry/react-native-track-player/issues/1647

File: ios/RNTrackPlayer/RNTrackPlayer.swift [hunk 1]
```diff
@@ -482,6 +468,7 @@ public class RNTrackPlayer: RCTEventEmitter, AudioSessionControllerDelegate {
         }
 
         player.stop()
+        player.reset()
```
Comment (mpivchev) [inline ios/RNTrackPlayer/RNTrackPlayer.swift:471]: This is a private function of SwiftAudio. Did you mean to put something else here? @dcvz

**Review by mpivchev — APPROVED** [review]
LGTM :)

**Pull request merged**